src/algoliasearch.helper.js

```diff
@@ -294,6 +295,9 @@ AlgoliaSearchHelper.prototype.searchForFacetValues = function(facet, query, maxF
   return searchForFacetValuesPromise.then(function addIsRefined(content) {
     self._currentNbQueries--;
     if (self._currentNbQueries === 0) self.emit('searchQueueEmpty');
+
+    content = isArray(content) ? content[0] : content;
```

Comment:
Array.isArray is sufficiently supported (IE8)

Reply:
It's IE9 but still ok to use it, since we support IE9+ with the helper.

Reply:
Fine here, but I saw there are some other usages of it, the lodash/isArray only does `module.exports = Array.isArray` anyway